tests/Integration/Presentation/Presenters/ConfirmSubscriptionHtmlPresenterTest.php

```diff
@@ -30,7 +31,7 @@ public function testGivenFailureResponse_templateIsRenderedWithoutMessages() {
 		$twig->expects( $this->once() )
 			->method( 'render' )
 			->with( [ 'error_message' => 'The confirmation code has expired.' ] );
-		$presenter = new ConfirmSubscriptionHtmlPresenter( $twig );
+		$presenter = new ConfirmSubscriptionHtmlPresenter( $twig, new Translator( 'en' ) );
```

Comment:
This case is a little bit misleading because the translator is not initialized with translations. It works none the less, because when it doesn't  find the message, it will return it unchanged. But to really have an "integration test" it could be initialized with an array like in the first code example of http://symfony.com/doc/current/components/translation/usage.html